PocketKit/Sources/Analytics/Tracker/PocketSnowplowTracker.swift

```diff
@@ -2,13 +2,13 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import SnowplowTracker
+@preconcurrency import SnowplowTracker
```

Comment:
We should open an issue on https://github.com/snowplow/snowplow-ios-tracker/issues

Reply:
done. I figure this will not be the only case...